src/hera/workflows/_mixins.py

```diff
@@ -449,7 +449,7 @@ class TemplateMixin(SubNodeMixin, HookMixin, MetricsMixin):
     name: Optional[str] = None
     node_selector: Optional[Dict[str, str]] = None
     parallelism: Optional[int] = None
-    http: Optional[HTTP] = None
+    http: Optional[HTTP] = None  # THIS SHOULD NOT BE HERE
```

Comment:
can we be a bit more detailed on this comment?

Reply:
Ah yes lol, forgot to add the reason why